graphql/schema.graphqls

```diff
@@ -140,6 +140,24 @@ type Mutation {
         """
         gentx: JSON!
     ): Void @auth
+
+    """
+    Emit a `RegisterRPCEndpointEvent` in the system to register RPC node url for the given druid validator.
+
+    Through the event handling logic, the validator will be fulfilled with his RPC endpoint url and task completed with points
+    attribution if still in progress. If event is submitted multiple time, RPC endpoint is updated.
+    """
+    registerRPCEndpoint(
+        """
+        The validator moniker that will register the RPC endpoint.
+        """
+        moniker: String!
```

Comment:
I think it would be better to identify the validator through its valoper address, the moniker is stable in the leaderboard but it may not be.

```suggestion
        valdator: ValoperAddress!
```